src/views/ScrollerMultipleChildren.js

```diff
@@ -9,13 +10,14 @@ const types = {
 	UiScrollerJS
 };
 
-const ScrollerMultipleChildren = ({location}) => {
+const ScrollerMultipleChildren = () => {
+	const location = useLocation();
 	const search = qs.parse(location.search, {ignoreQueryPrefix: true});
 	const type = search.type;
 	const Scroller = types[type] || ScrollerJS;
 
 	return (
-		<Scroller id="Scroller" animate>
+		<Scroller id="Scroller">
```

Comment:
*Note: 'animate' prop was deleted as it caused a console error: "'Received "true" for a non-boolean attribute". 'animate' is not a prop of Scroller, it's a parameter for the 'scrollTo' function, so we believe it is not needed here.